specs/SensitivityLabel.md

```diff
@@ -0,0 +1,541 @@
+  /// | `*` | `https://any-site.com` | Yes | Wildcard matches all URLs |
+  ///
+  /// Setting the allowlist to an empty array will disable access to the
+  /// PageInteractionRestrictionManager API for all pages.
+  ///
+  /// Changes take effect immediately for all WebView2 instances using this profile.
+  /// The allowlist is persisted across sessions.
+  HRESULT SetPageInteractionRestrictionManagerAllowlist(
+      [in] UINT32 allowlistCount,
+      [in] LPCWSTR* allowlist
+  );
+}
+```
+### .NET/WinRT
+```idl
+namespace Microsoft.Web.WebView2.Core
+{
+    runtimeclass CoreWebView2Profile
+    {
+        /// Controls which URLs are allowed to access the PageInteractionRestrictionManager API.
+        /// 
+        /// This property manages an allowlist of URLs that determines which web pages
+        /// can use the PageInteractionRestrictionManager API. Only URLs that match 
+        /// entries in this allowlist (either exact matches or wildcard patterns) will
+        /// have access to the PageInteractionRestrictionManager functionality.
+        /// 
+        /// The allowlist accepts both exact URL strings and wildcard patterns.
+        /// For wildcard patterns, `*` matches zero or more characters.
+        /// 
+        /// | URL Filter | Page URL | Access Granted | Notes |
+        /// | ---- | ---- | ---- | ---- |
+        /// | `https://example.com` | `https://example.com/page` | No | Exact match required |
+        /// | `https://example.com` | `https://example.com` | No | The URI is normalized before filter matching so the actual URI used for comparison is https://example.com/ |
+        /// | `https://example.com/*` | `https://example.com/page` | Yes | Wildcard matches any path |
+        /// | `*://example.com/*` | `https://example.com/page` | Yes | Wildcard matches any scheme |
+        /// | `*` | `https://any-site.com` | Yes | Wildcard matches all URLs |
+        IVectorView<String> PageInteractionRestrictionManagerAllowlist { get; set; };
+    }
+}
+```
+
+## Sensitivity label change event
+### C++ 
+```
+/// Represents the state of sensitivity label detection and processing
+/// for web page loaded in the WebView2 control. This enum indicates
+/// whether sensitivity labels have been detected, are being processed,
+/// or are fully determined for the current web page content.
+[v1_enum]
+typedef enum COREWEBVIEW2_SENSITIVITY_LABEL_STATE {
+  /// Indicates that none of the loaded pages are in the allow list. Hence 
+  /// none will report sensitivity labels. 
+  COREWEBVIEW2_SENSITIVITY_LABEL_STATE_NONE,
+  /// Indicates that WebView2 has loaded pages from the allow list that can
+  /// report sensitivity labels, but the label determination is not yet 
+  /// complete.
+  COREWEBVIEW2_SENSITIVITY_LABEL_STATE_UNDETERMINED,
+  /// Indicates that WebView2 has loaded pages from the allow list,
+  /// and those pages have provided label information.
+  COREWEBVIEW2_SENSITIVITY_LABEL_STATE_DETERMINED,
```

Comment:
Please update comments to say this means we have at least one web page reporting.

Please rename DETERMINED -> AVAILABLE